src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -612,7 +618,6 @@ protected override void OnElementChanged(ElementChangedEventArgs<Frame> e)
                 CaptureCallback = new CameraCaptureListener(this);
                 imageAvailableListener = new ImageAvailableListener(this);
                 surfaceTextureListener = new Camera2BasicSurfaceTextureListener(this);
-                StartBackgroundThread();
```

Comment:
Is this no longer required?

Reply:
It's now happening only in the OnActivityResume (it was executed twice)